addons/purchase/static/src/product_catalog/kanban_record.js

```diff
@@ -36,8 +36,7 @@ patch(ProductCatalogKanbanRecord.prototype, {
         const price = await rpc("/product/catalog/update_order_line_info", {
             order_id: this.env.orderId,
             product_id: this.env.productId,
-            product_packaging_id: packaging.id,
-            product_packaging_qty: productPackagingQty,
+            packaging_uom_qty: productPackagingQty,
```

Comment:
I'm not sure we even need this custom packaging qty logic for the PO catalog anymore. The existing logic should be able to handle different `product_id.uom_id` vs `pol.product_uom_id`